src/Filesystem/Exception/UnableToCreateDirectory.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Filesystem\Exception;
+
+use RuntimeException;
+
+class UnableToCreateDirectory extends RuntimeException
+{
+    public static function atLocation(string $location, string $because): self
```

Comment:
What's the difference between `location` and `path` in the previous exception?

Reply:
`$path` is where I started, `$location` is what I am refactoring to.